DirectProgramming/DPC++FPGA/Tutorials/Features/fp_pragmas/sample.json

```diff
@@ -0,0 +1,55 @@
+{
+  "guid": "7909FAE1-D3D4-4E97-A963-14A884F33495",
```

Comment:
Please confirm you followed the procedure to generate a unique guid for this tutorial, and did not simply copy and paste this from a different tutorial.